src/Microsoft.DotNet.Build.Tasks/PackageFiles/msbuild.cmd

```diff
@@ -0,0 +1,8 @@
+@if "%_echo%" neq "on" echo off
```

Comment:
Looks like this cmd file can be condensed some more. I don't think we need the setlocal or the if ERRORLEVEL check just exit with ERRORLEVEL